components/Actions.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import { View, StyleSheet } from 'react-native'
+import { IconButton } from 'react-native-paper'
+
+import theme from '../defaultTheme'
+
+const A11Y_LABELS = {
+  code: 'Add new secret by scanning a QR Code',
+  upload: 'Add new secret by uploading a QR Code',
+  add: 'Add new secret by filling the details',
+}
+
+export default function Actions() {
+  return (
+    <View style={styles.actions}>
+      <IconButton
+        icon="qrcode"
+        accessibilityLabel={A11Y_LABELS.code}
+        mode="contained"
+        style={styles.button}
+        size={40}
+        color={theme.colors.white}
+      />
+      <IconButton
+        icon="upload"
+        accessibilityLabel={A11Y_LABELS.upload}
+        mode="contained"
+        style={styles.button}
+        size={40}
+        color={theme.colors.white}
```

Comment:
this is more of a semantic issue. not sure this is a rule we can apply to this project because I did not look closely at the designs, so please take is just as a consideration. if we want our UI elements to have this color, this should be the primary color in the theme.